packages/replay/test/integration/sendReplayEvent.test.ts

```diff
@@ -146,13 +148,13 @@ describe('Integration | sendReplayEvent', () => {
     expect(replay.eventBuffer?.pendingLength).toBe(0);
   });
 
-  it('uploads a replay event if 15 seconds have elapsed since the last replay upload', async () => {
+  it('uploads a replay event if maxFlushDelay is set 15 seconds have elapsed since the last replay upload', async () => {
```

Comment:
Hmm, any clue why this test wasn't failing before? 😅 I suspect there is actually a bug (??) that is fixed here, but not sure - we do have a test for the wait time being equal, but maybe it doesn't catch some aspect...?!